sentry/sentry_test.go

```diff
@@ -0,0 +1,96 @@
+package sentry
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-wit/account"
+	"github.com/fabric8-services/fabric8-wit/login/tokencontext"
+	testtoken "github.com/fabric8-services/fabric8-wit/test/token"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/getsentry/raven-go"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
+	"github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+)
+
+func Test_extractUserInfo(t *testing.T) {
+
```

Comment:
Add resource.Require(t, resource.UnitTest)

Reply:
added this, thanks :-)